Frontend/ui-common/src/Components/Banner/View.purs

```diff
@@ -32,63 +34,53 @@ view push config =
     [ height WRAP_CONTENT
     , width MATCH_PARENT
     , cornerRadius 12.0
-    , margin $ MarginTop 12
     , background config.backgroundColor
     , visibility if config.isBanner then VISIBLE else GONE
     , onClick push (const OnClick)
     ]
     [  linearLayout
-        [ height WRAP_CONTENT
+        [ height MATCH_PARENT
         , weight 1.0
-        , padding $ Padding 20 7 0 0
+        , padding if (os == "IOS") then Padding 20 16 0 18 else Padding 20 13 0 18
         , orientation VERTICAL
+        , gravity CENTER_VERTICAL
         ]
         [ textView
           [ height WRAP_CONTENT
           , width MATCH_PARENT
           , gravity LEFT
           , text config.title
           , color config.titleColor
-          , textSize FontSize.a_16
           , fontStyle $ FontStyle.bold LanguageStyle
+          , textSize if(screenWidth unit < 380) then  FontSize.a_14 else  FontSize.a_15
           ]
         , linearLayout
           [ height WRAP_CONTENT
           , width WRAP_CONTENT
           , gravity CENTER_VERTICAL
+          , if (os == "IOS") then margin (MarginTop 4) else margin (MarginTop 0)
```

Comment:
@prakharritik add the check inside MarginTop instead of adding it at the beginning.

Reply:
Done